package-name-map/src/test/package-name-map_test.ts

```diff
@@ -0,0 +1,226 @@
+/**
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import chai = require('chai');
+
+import {PackageNameMap, isPathSegmentPrefix} from '../package-name-map.js';
+
+const {assert} = chai;
+
+suite('PackageNameMap', () => {
+  suite('resolve', () => {
+    const referrerURL = 'http://foo.com/';
```

Comment:
Maybe the referrerURL should be something more like `https://foo.com/js/entry.js` for a more realistic starting point.

Reply:
done